litgpt/model.py

```diff
@@ -162,34 +171,40 @@ def forward(
     ) -> torch.Tensor:
         """
         Non-parallel residual       Parallel residual
-           ┌─ x                     ┌─ x ────────────┐             Note: if `shared_attention_norm` is True,
-           │  ↓                     │  ↓             ↓                   the output from `norm_1` is reused
-           │  norm_1                │  norm_1  ───►  norm_2
-           │  ↓                     │  ↓             ↓
-           │  attn                  │  attn          mlp
-           │  ↓                     │  ↓             │
-        ┌─ └► +                     └► + ◄───────────┘
+           ┌─ x                     ┌─ x ──────────────────┐             Note: if `shared_attention_norm` is True,
```

Comment:
Thanks for updating this. This is a really nice go-to drawing to also explain pre and post LN